bin/p2-exec/main.go

```diff
@@ -29,6 +29,7 @@ var (
 	nolim          = kingpin.Flag("nolimit", "Remove rlimits.").Short('n').Bool()
 	clearEnv       = kingpin.Flag("clearenv", "Clear all environment variables before loading envDir(s).").Bool()
 	workDir        = kingpin.Flag("workdir", "Set working directory.").Short('w').String()
+	umask          = kingpin.Flag("umask", "umask for this process and its subprocesses. Use golang octal notation ex. 022").Short('m').Default("0022").Int()
```

Comment:
Octal notation is pretty standard these days; I wouldn't call it the "golang notation."